### PR TITLE
[14.0] [MIG] hr_employee_birth_name

### PR DESCRIPTION
**Pull request by Chanakya-OSI**

### CODE REVIEW

File: hr_employee_birth_name/readme/INSTALL.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+To install this module, you need to:
+
+1.  Clone the branch 11.0 of the repository https://github.com/OCA/hr
```
Comment (patrickrwilson) [inline hr_employee_birth_name/readme/INSTALL.rst:3]: This line should be updated, perhaps something a bit more general and not pointing to the specific version. Small detail, not a stopper.

File: hr_employee_birth_name/views/hr_employee_view.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" ?>
```
Comment (patrickrwilson) [inline hr_employee_birth_name/views/hr_employee_view.xml:1]: This line is no longer needed since v11. Cosmetic only.

**Review by patrickrwilson — APPROVED** [review]
Small suggestions, not blocking though. Tested and LGTM

**Comment by Saran440** [comment]
@Chanakya-OSI Thank you for your contribution.
Please, Could you rebase your PR into 1 commit?

**Comment by patrickrwilson** [comment]
@osi-scampbell Can you help review this?

**Review by osi-scampbell — APPROVED** [review]

File: hr_employee_birth_name/README.rst [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+.. contents::
+   :local:
+
+Installation
+============
+
+To install this module, you need to:
+
+1.  Clone the branch 11.0 of the repository https://github.com/OCA/hr
```
Comment (olgamarcocb) [inline hr_employee_birth_name/README.rst:42]: Can you remake the README please. I think this installation is wrong and no necessary :wink:

**Review by olgamarcocb — APPROVED** [review]
Fine for me if the README is remade

**Pull request merged**